web3/contract.py

```diff
@@ -779,6 +803,8 @@ class ConciseContract:
 
     > contract.functions.withdraw(amount).transact({'from': eth.accounts[1], 'gas': 100000, ...})
     '''
+    method_class = ConciseMethod
```

Comment:
A key point of the refactor was to reduce/remove name conflicts. This adds another potential conflict. But I think we can get to your goal in almost the same way. (see following comments for pseudocode)

Reply:
I was thinking about that way too, this way was just easier so I figured I'd give it a shot